Dockerfiles/manifests/cluster-agent/hpa-example/cluster-agent-hpa-svc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: datadog-cluster-hpa
```

Comment:
I think this should be called `datadog-custom-metrics-server` or `datadog-metrics-server` since this is the service that extends the apiserver.